auth-api/src/auth_api/services/authorization.py

```diff
@@ -164,8 +163,11 @@ def as_dict(self, expanded: bool = False):
 
     @staticmethod
     def _is_product_based_auth(product_code):
+
         check_product_based_auth = False
         if product_code:
+            from auth_api.services.products import \
```

Comment:
why is it needed here? I dont see any usage

Reply:
cyclic import. now authorization uses products and vice versa